.travis.yml

```diff
@@ -3,8 +3,18 @@ os:
   - linux
   - osx
 node_js:
+  - "8.11.3"
   - "8"
   - "10"
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.12.3
```

Comment:
Make sure the version of yarn being used in CI is lowest version we support per `"engines"` of embark/-ui's `package.json`.